src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java

```diff
@@ -181,6 +200,14 @@ private void checkParagraphTag(DetailNode tag) {
         else if (newLine == null || tag.getLineNumber() - newLine.getLineNumber() != 1) {
             log(tag.getLineNumber(), MSG_LINE_BEFORE);
         }
+
+        final BlockTagViolation blockTag = checkIfFollowedByBlockTag(tag);
+
+        if (blockTag != null) {
```

Comment:
Please remove this empty line

Reply:
done 

https://github.com/checkstyle/checkstyle/blob/03f5d5ecffd7969cf64ab60ab12811ee8ce3d1d0/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java#L204-L207